nvflare/app_common/workflows/base_model_controller.py

```diff
@@ -379,6 +376,13 @@ def sample_clients(self, num_clients: int = None) -> List[str]:
 
         return clients
 
+    def set_fl_context(self, data: FLModel):
+        """Set up the fl_ctx information based on the passed in FLModel data."""
+        if data and data.current_round is not None:
+            self.fl_ctx.set_prop(AppConstants.CURRENT_ROUND, data.current_round, private=True, sticky=True)
+        else:
```

Comment:
Since this is now general purpose, I'm not sure this warning is needed. The workflow may not always be round-based ...

Reply:
That's the thing - there will be a lot of warning messages. Maybe we simply don't log any warning messages because they could be misleading.  Or maybe just DEBUG level.

Reply:
Changed to DEBUG message.